server/src/main/resources/db/migration/V0002__create_certificate_tables.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE certificate
+(
+    certificate_id               serial PRIMARY KEY,
+    certificate_name_fi          text          NOT NULL,
+    certificate_content_fi       text          NOT NULL,
+    certificate_publish_state    publish_state NOT NULL,
+    certificate_created_at       timestamptz   NOT NULL default now(),
+    certificate_updated_at       timestamptz   NOT NULL default now(),
+    certificate_file_name        text          NOT NULL,
+    certificate_file_key         text          NOT NULL,
+    certificate_file_upload_date timestamptz   NOT NULL default now()
```

Comment:
Kannattaisikohan liitetiedostoille olla oma taulu näiden `file`-alkuisten kolumnien sijaan, ja tänne tulis foreign key siihen tauluun, vaik nimeltä `certificate_attachment`? Tästä vois olla ainaki pari etua:
- erillinen taulu paketois tiedostoon liittyvät kolumnit nättiin pakettiin, joita myös tulevat liitetiedostotarpeet (esim. ohjeet) vois hyödyntää. Näin kaikki liitetiedostotuet tulis automaattisesti toteutettua samalla tavalla.
- Joustavuus: jos päätetäänkin, että pitää tukea useaa liitetiedostoa todistuksissa, niin sen toteuttaminen on yksinkertaisempaa erillisellä taululla.

Esim. meillä vois olla `attachment`-taulu, jossa on kaikille erilaisille liitetiedostoille yhteiset kolumnit. Alkuun varmaan nää kolme `file`-alkuista kolumnia. Sit vaik `certificate_attachment` vois periä `attachment`-taulun, ja jos certti-liitetiedostoihin ikinä tulee jotain spessu-kolumneja, niin ois helppo lisätä niitä.